src/modules/shared/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './Popup.module.scss';
+
+export const Popup: React.FC = () => {
+  const { t } = useTranslation();
+  const [isVisible, setIsVisible] = useState(false);
+  const [email, setEmail] = useState('');
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setIsVisible(true);
+    }, 10000);
```

Comment:
Please use constant here. `const SUBSCRIBE_DELAY = 10000` and move it to folder constants in index.ts (already exists)